Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHG_Output_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+OutALCARECOPromptCalibProdSiPixelAliHG_noDrop = cms.PSet(
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOPromptCalibProdSiPixelAliHG')
+    ),
+    outputCommands = cms.untracked.vstring(
+        'keep *_SiPixelAliMillePedeFileConverterHG_*_*')
+)
+
+import copy
+
+OutALCARECOPromptCalibProdSiPixelAliHG=copy.deepcopy(OutALCARECOPromptCalibProdSiPixelAliHG_noDrop)
```

Comment:
I know it's a common problem of all the other alcarecos, but `clone` is preferred to `copy.deepcopy`. 
Can you changes it here?